ci/kokoro/docker/build-in-docker-cmake.sh

```diff
@@ -115,6 +122,19 @@ ${CMAKE_COMMAND} \
 echo
 log_yellow "Finished CMake config"
 
+if [[ "${CLANG_TIDY:-}" == "yes" && (\
+  "${KOKORO_JOB_TYPE:-}" == "PRESUBMIT_GITHUB" || \
+  "${KOKORO_JOB_TYPE:-}" == "PRESUBMIT_GIT_ON_BORG") ]]; then
+  # For presubmit builds we only run clang-tidy in the files that have changed
+  # w.r.t. the target branch.
+  echo
+  log_yellow "Running clang-tidy on presubmit build, only changed files are tested."
+  git diff --name-only "${KOKORO_GITHUB_PULL_REQUEST_TARGET_BRANCH:-${BRANCH}}" |
+    grep -E '\.(cc|h)$' |
+    xargs -d '\n' -r -P "${NCPU}" clang-tidy -p="${BINARY_DIR}"
```

Comment:
The xargs man page says 

> Use the -n option or the -L option with -P; otherwise chances are that only one exec will be done. 

In a PR with 10s of edited files, xargs will likely pack all of those into a single clang-tidy invocation thus defeating the parallelism. You might want something like `-n 1 -P "${NCPU}"` to maximize parallelism.

Reply:
Done.